src/components/business/DeleteIdentity.tsx

```diff
@@ -32,8 +34,14 @@ export const DeleteIdentity: React.FC<DeleteIdentityProps> = ({
     value: value,
   };
 
-  const deleteIdentity = (): Promise<Response> => {
-    return fetchJson("/api/account", HttpVerbs.DELETE, requestData);
+  const deleteIdentity = async (): Promise<void> => {
+    return fetchJson("/api/account", HttpVerbs.DELETE, requestData)
+      .then(() => {
+        router && router.push("/account/logins/");
+      })
+      .catch((error: Error) => {
+        console.warn(error.message);
+      });
```

Comment:
You should throw an Error instead of a console.warn.

Reply:
OK fixed e5efe15